.github/workflows/coverity.yml

```diff
@@ -47,31 +47,74 @@ permissions:
   contents: read
 
 jobs:
-  scan:
-    runs-on: ubuntu-latest
+  coverity-scan:
+    runs-on: ubuntu-22.04
     if: ${{ github.repository_owner == 'apache' }}
 
-    env:
-      DEBIAN_FRONTEND: noninteractive
+    container:
+      image: apache/incubator-cloudberry:cbdb-build-rocky9-latest
+      options: >-
+        --user root

```

Comment:
why use the root user ?

Reply:
Hi @jiaqizho I take this module as a reference: https://github.com/apache/cloudberry/blob/f18593370b2b991823080916b46a4b7cb173769f/.github/workflows/build-cloudberry.yml#L363